packages-ts/starknet-gauntlet-multisig/package.json

```diff
@@ -25,7 +25,7 @@
     "bundle": "yarn build && pkg ."
   },
   "dependencies": {
-    "starsign-multisig": "0.2.3",
+    "starsign-multisig": "0.3.0",
```

Comment:
This bump should wait for cairo 0.10 support since `starsign-multisig` updated their code https://github.com/eqlabs/starknet-multisig/commit/3eef643f14b82c41a97778f1d190b3b6f8cf0bd8